app/routes/__docs/$/$.tsx

```diff
@@ -58,7 +59,10 @@ export const loader: LoaderFunction = async ({ params, request }) => {
       data.collectionRootPath,
       page.origin.relativePath
     )
-    const pageBasePath = nodePath.posix.dirname(path) + "/"
+    const pageBasePath =
```

Comment:
Maybe we should rename this? It's not really a "base path" anymore -- or at least it's more confusing now. Because it's actually the path to the full page now. 

Is there even a need to extract the `dirname` and `basename` and piece them back together? I think this whole thing is just equivalent to `const pageBasePath = stripExtension(path)` now, isn't it?

Reply:
This makes sense to me. 
The only thing I'm unclear about is why were we separating things in the first place?

Reply:
So many possibilities. lol.